xbmc/cores/VideoRenderers/OverlayRenderer.cpp

```diff
@@ -328,12 +328,27 @@ COverlay* CRenderer::Convert(CDVDOverlaySSA* o, double pts)
 {
   CRect src, dst;
   g_renderManager.GetVideoRect(src, dst);
-
-  int width  = MathUtils::round_int(dst.Width());
-  int height = MathUtils::round_int(dst.Height());
-
+  RESOLUTION_INFO res = g_graphicsContext.GetResInfo();
+
+  int frameWidth = res.iWidth;
```

Comment:
screen is not correct either because it defines the area of the entire screen. if I understand this code right, you mean the target rendering area. Note that this may be different to the window area which you get with g_graphicsContext.GetResInfo(). The video control may not occupy the entire window.